extensions/ql-vscode/src/run-queries.ts

```diff
@@ -212,7 +239,28 @@ export class QueryEvaluationInfo {
             })
 
             .catch(err => {
-              void showAndLogWarningMessage(`Failed to generate structured evaluator log summary. Reason: ${err.message}`);
+              void showAndLogWarningMessage(`Failed to generate human-readable structured evaluator log summary. Reason: ${err.message}`);
+            });
+
+          // REVIEW: it seems clunky to have to run this command again just to get the same summary file
+          // in JSON format, but I'm not sure it's worth changing the way it works in the CLI for just
+          // this use case? 
+          void qs.cliServer.generateJsonLogSummary(this.evalLogPath, this.jsonEvalLogSummaryPath)
+            .then(() => {
+              // Convert summary into appropriate JSON object here. Perhaps move the parsing logic
+              // into another class later on?
+              
+              // REVIEW: We may want to stream this later on; we are generating the human-readable
+              // file in memory above but the JSON formatted one should be larger?
```

Comment:
After discussion with @adityasharad, decided to stream in a follow-up PR given this will go in under canary flag and make this PR ready for review. I have documented this as a TODO in the code comments.